FirebaseSessions/Sources/SessionInitiator.swift

```diff
@@ -89,11 +89,11 @@ class SessionInitiator {
     #endif
   }
 
-  @objc func appBackgrounded() {
+  @objc private func appBackgrounded() {
```

Comment:
Thanks for restricting the access to this API. Curious if we need to do such an audit to the rest of the library to make sure we are drawing restrictions around classes clearly? Not a blocking comment - but something for following up.

Reply:
Yeah we should totally do it - let me add a task